.github/workflows/trivy_scan.yml

```diff
@@ -12,6 +12,10 @@ on:
         description: 'The released Antrea version to scan'
         type: string
         required: false
+      no-cache:
```

Comment:
This is intentional actually, because the workflow needs to run both for `schedule` as well as `workflow_dispatch`. When it runs as a scheduled workflow, the inputs are not set, so `no-cache` is empty. The intent is also to have caching be the default behavior.
After that, `${{ inputs.no-cache }}` means that caching was explicitly disabled, and `${{ !inputs.no-cache }}` means that caching was not explicitly disabled, which works well.
When using `cache` instead of `no-cache`, the if conditions become more complicated and error prone.
See also https://github.com/orgs/community/discussions/29242

Reply:
Ah, got it, thanks for the explanation. 👍